flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionTest.java

```diff
@@ -148,15 +148,17 @@ public void testAddOnReleasedBlockingPartition() throws Exception {
 	/**
 	 * Tests {@link ResultPartition#addBufferConsumer} on a partition which has already been released.
 	 *
-	 * @param pipelined the result partition type to set up
+	 * @param partitionType the result partition type to set up
 	 */
-	protected void testAddOnReleasedPartition(final ResultPartitionType pipelined)
+	protected void testAddOnReleasedPartition(final ResultPartitionType partitionType)
```

Comment:
can be private, also some other methods in this class

Reply:
ditto